howtos/weighting_scheme.rst

```diff
@@ -45,6 +85,29 @@ The UnigramLM weighting formula is based on an original approach by Bruce Croft.
 It uses statistical language modelling; 'unigram' in this case means that
 words are considered to occur independently.
 
+The Dirichlet prior method is one of the best performing language modeling approaches. Xapian
+now provides support for a modified Dirichlet prior method, namely Dir+ which is an improvement over
+the original Dirichlet prior method as it is particularly more effective across web collections
```

Comment:
I'd go with just "the original at it is more effective…" here — reading "Dirichlet prior method" repeatedly just slows things down :-).